src/pytest_plugins/consume/hive_simulators/ruleset.py

```diff
@@ -336,6 +342,9 @@
         "HIVE_TERMINAL_TOTAL_DIFFICULTY": 0,
         "HIVE_SHANGHAI_TIMESTAMP": 0,
         "HIVE_CANCUN_TIMESTAMP": 0,
+        "HIVE_CANCUN_BLOB_TARGET": 3,
```

Comment:
Just wondering why Cancun config is necessary here? Because clients starting after the Prague fork still require this config to sync Cancun blocks? So it'll always be expected?